src/test/python/MCSVerification.py

```diff
@@ -568,14 +679,26 @@ def main(argv):
     ssDir = workingDir + '/scisparkGTG'
 
     try:
-        opts, _ = getopt.getopt(argv, "h")
-        if len(opts) == 1:
-            if '-h' in opts[0][0]:
+        opts, arg = getopt.getopt(argv, "hs:")
+
+        if len(opts) == 0:
+            print 'Please run: python MCSVerification.py -h'
+            sys.exit(2)
+
+        for opt, arg in opts:
+            if '-h' in opt: 
                 print 'These are the verification test to determine if SciSpark GTG is operating correctly.'
+                print 'To run: python MCSVerification.py -s scalaVersion'
+                print 'For example: python MCSVerification.py -s scala-2.11'
                 print 'The data used in the runs were used in the paper, and the python GTG results are downloaded.'
                 print 'If needed, the original GTG can de downloaded and used to generate the results. \
                     Please use the original release - Release 1.0'
                 sys.exit(2)
+            elif '-s' in opt: 
+                scalaVersion = arg
+                if not os.path.exists('../../../target/' + scalaVersion):
```

Comment:
Currently the user has to run this code from the src/test/python directory or else all path references break.

Do you think it may be of any use to allow the user to run this script from anywhere i.e make paths used in this code relative to where the script resides and not from where it is executed ? Maybe we could solve this by setting the workingDir differently. 

Thoughts @kwhitehall